compass/ocean/tests/spherical_harmonic_transform/QU_convergence/__init__.py

```diff
@@ -0,0 +1,73 @@
+import configparser
+
+from compass.config import add_config
+from compass.testcase import TestCase
+
+from compass.ocean.tests.spherical_harmonic_transform.QU_convergence.mesh import Mesh
+from compass.ocean.tests.spherical_harmonic_transform.QU_convergence.init import Init
+#from compass.ocean.tests.spherical_harmonic_transform.QU_convergence.analysis import Analysis
+
+
+class QUConvergence(TestCase):
+    """
+    A test case for creating a global MPAS-Ocean mesh
+    Attributes
+    ----------
+    resolutions : list of int
+    """
+    def __init__(self, test_group):
+        """
+        Create test case for creating a global MPAS-Ocean mesh
+        Parameters
+        ----------
+        test_group : compass.ocean.tests.cosine_bell.GlobalOcean
+            The global ocean test group that this test case belongs to
+        """
+        super().__init__(test_group=test_group, name='QU_convergence')
+        self.resolutions = None
+
+    def configure(self):
+        """
+        Set config options for the test case
+        """
+        config = self.config
+        resolutions = config.get('QU_convergence', 'resolutions')
+        resolutions = [int(resolution) for resolution in
+                       resolutions.replace(',', ' ').split()]
+
+        self.resolutions = resolutions
+
+        serial_nLat = config.get('QU_convergence','serial_nLat')
+        serial_nLat = [int(nLat) for nLat in 
+                       serial_nLat.replace(',', ' ').split()]
+
+        parallel_N = config.get('QU_convergence','parallel_N')
+        parallel_N = [int(N) for N in 
+                       parallel_N.replace(',', ' ').split()]
+
+        #init_options = dict()
+        #for option in ['function_option','n_iterations']:
+        #    init_options['config_test_sht_{}'.format(option)] = \
+        #        config.get('QU_convergence', option)
+
+        for resolution in resolutions:
+            self.add_step(Mesh(test_case=self, resolution=resolution, serial_nLat=serial_nLat))
+
+            for N in parallel_N:
+    
+                self.add_step(Init(test_case=self, resolution=resolution, algorithm='parallel', order=N))
+
+            for nLat in serial_nLat:
+    
+                self.add_step(Init(test_case=self, resolution=resolution, algorithm='serial', order=nLat))
+
+        #self.add_step(Analysis(test_case=self, resolutions=resolutions))
+
+
+
+    def run(self):
+        """
+        Run each step of the testcase
+        """
+        # run the step
+        super().run()
```

Comment:
This should just be removed unless you plan to add to this, since the superclass' `run()` will get called if this class doesn't override it.